.github/workflows/release.yml

```diff
@@ -5,141 +5,246 @@ on:
       - 'v*'
 
 jobs:
-  build-binary:
-    name: Build binary
+
+  build-package:
+    name: Build package
     runs-on: ubuntu-latest
+    container:
+      image: ubuntu:16.04
```

Comment:
Why not specify `runs-on` to the Ubuntu version you want?

Reply:
There we need to keep this, because we want to compile the code in a unified and clean system